classes/metadata-generator.bbclass

```diff
@@ -15,7 +15,7 @@ def create_dep(id, required_version, min_version=None, max_version=None):
     return {FIELD_ID: id, FIELD_REQUIRED_VERSION: required_version, FIELD_MIN_VERSION: min_version, FIELD_MAX_VERSION: max_version}
 
 
-def create_component_metadata(id, file_name, vendor_version, description=None, install_dep=None, runtime_deps=None, annotations=None):
+def create_component_metadata(id, file_name, vendor_version, description=None, install_dep=None, runtime_deps=None, annotations=None, download_ttl=None):
```

Comment:
First line of commit message <= 72 chars

Reply:
updated